corehq/apps/builds/templates/builds/bootstrap5/edit_menu.html

```diff
@@ -15,103 +15,89 @@
 <div class="alert alert-info"><strong>{% trans "Editing" %} {{ doc.ID }}</strong></div>
 
 
-<div class="card card-default">
-  <div class="card-header">
-    <h3 class="card-title">{% trans "Menu Options" %}</h3>
-  </div>
+<div class="card">
```

Comment:
```suggestion
<div class="card mb-4">
```

which means "margin bottom 4"

Reply:
suggestion applies elsewhere

Reply:
82337d34556